doc/modules/mod_extdisco.md

```diff
@@ -0,0 +1,36 @@
+### Module Description
+Implements [XEP-0215: External Service Discovery](http://xmpp.org/extensions/xep-0215.html) for discovering information about services external to the XMPP network.
+The main use-case is to help discover STUN/TURN servers to allow for negotiating media exchanges.
+
+### Options
+* **type** (atom, default: unset) - service type
+
+    * **Valid values:** `stun, turn`
+
+#### Service Options
+* **host** (string, required, default: unset): Hostname or an IP address where the service is hosted.
```

Comment:
If it is a required option why fo you provide a default value? Let it crush

Reply:
The default value is not set.